pages/common/tokei.md

```diff
@@ -0,0 +1,19 @@
+# tokei
+
+> A program that allows you to count your code, quickly.
```

Comment:
What does it count, precisely?

Reply:
That's the official short description 🙂

Looks like it counts Lines of Code, number of files, and a few other things.

Reply:
Ah, ok! In that case,  think this is probably accurate enough :P

Reply:
I am still not happy with this. Can we change it to "A program that displays statistics about one's code" ?